dspace-server-webapp/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java

```diff
@@ -105,60 +116,60 @@ public void setUp() throws Exception {
         context.turnOffAuthorisationSystem();
 
         parentCommunity = CommunityBuilder.createCommunity(context)
-                                          .withName("Parent Community")
-                                          .build();
+                .withName("Parent Community")
+                .build();
```

Comment:
@benbosman : Just a quick note that this PR needs a rebase & cleanup.  It looks like nearly all the changes to `RelationshipRestRepositoryIT` are **space changes** (where the IDE seems to have made a lot of unrelated alignment changes).  If you could revert those space changes it'd make this PR easier to review (and it might even fall into the "1 approval" category).